locales/nl_NL.json

```diff
@@ -1,109 +1,109 @@
+      "confirm-button-label": "Trek alle sleutels voor altijd in",
+      "cancel-button-label": "Ga terug"
     },
     "delete-token": {
-      "body": "Are you sure you want to revoke ‘<0></0>’?",
-      "confirm-button-label": "Revoke forever",
-      "cancel-button-label": "Go back"
+      "body": "Ben je zeker dat je wilt intrekken ‘<0></0>’?",
+      "confirm-button-label": "Trek voor altijd in",
+      "cancel-button-label": "Ga terug"
     },
     "edit-whitelisted": {
-      "title": "Address whitelist for {{name}}",
-      "add": "Add another",
-      "save-and-close": "Saves changes and close",
-      "continue": "Continue"
+      "title": "Adres gewhitelist voor {{name}}",
+      "add": "Nog één toevoegen",
+      "save-and-close": "Veranderingen opslaan en sluiten",
+      "continue": "Ga verder"
     },
     "overview": {
-      "no-addresses-whitelisted": "No addresses whitelisted",
-      "title": "Manage API keys",
-      "revoke-button-label": "Revoke",
-      "table-header": "Your API keys",
-      "no-tokens-exp": "<0>Generate your first key</0> or read our <1>API documentation</1>.",
-      "name": "Name",
-      "never-used": "Never used",
-      "last-used": "Last used {{when}}",
-      "subtitle": "Generate and manage API keys. Learn more about <0>our APIs</0>.",
-      "key-info": "Key information",
-      "api-key": "API key",
+      "no-addresses-whitelisted": "Geen adressen gewhitelist",
+      "title": "Beheer API-sleutels",
+      "revoke-button-label": "Intrekken",
+      "table-header": "Jouw API-sleutels",
+      "no-tokens-exp": "<0>Genereer jouw eerste sleutel</0> of lees onze <1>API documentation</1>.",
+      "name": "Naam",
+      "never-used": "Nooit gebruikt",
+      "last-used": "Laatst gebruikt {{when}}",
+      "subtitle": "Genereer en beheer API-sleutels. Leer meer over <0>our APIs</0>.",
+      "key-info": "Sleutel informatie",
+      "api-key": "API-sleutel",
       "blockchains": "Blockchains",
-      "whitelisted": "Whitelist",
-      "view-whitelist": "View whitelist",
+      "whitelisted": "Witte lijst",
+      "view-whitelist": "Bekijk witte lijst",
       "blockchains-static": "Bitcoin, Ethereum, NEO",
-      "blockchains-hint": "This API key can function with these blockchains.",
-      "usable-for": "Usable for",
-      "revoke": "Revoke",
-      "created": "Created",
-      "policy-trading": "Trading",
-      "loading": "Loading tokens…",
-      "no-tokens": "You haven’t generated any API keys yet.",
-      "policy-withdrawals": "Withdrawals and deposits",
-      "revoke-all-keys-button-label": "Revoke all keys",
-      "generate-new-key-button-label": "Generate new key",
+      "blockchains-hint": "Deze API-sleutel werkt met deze blockchains.",
+      "usable-for": "Bruikbaar voor",
+      "revoke": "Intrekken",
+      "created": "Gecreëerd",
+      "policy-trading": "Handelen",
+      "loading": "Tokens worden gelaad…",
```

Comment:
Ik denk dat "gelaad" niet het goeie woord ervoor is.  "Tokens worden geladen","Laden tokens" of "Bezig met het laden van de tokens".

Reply:
Wat denk je van "Laden van tokens…" ?

Reply:
"Laden van tokens" klinkt goed.